pkg/volume/awsebs/attacher.go

```diff
@@ -192,6 +201,44 @@ func (attacher *awsElasticBlockStoreAttacher) WaitForAttach(spec *volume.Spec, d
 	}
 }
 
+// getDiskNumber gets the Windows disk number for a given volume ID. The disk number is needed for mounting.
+// TODO This only works for Nitro-based instances
+// TODO fallback to Get-Disk
+func (attacher *awsElasticBlockStoreAttacher) getDiskNumber(volumeID string) (string, error) {
+	// Split the ID from zone: aws://us-west-2b/vol-06d0909eb358b05f9
+	split := strings.Split(volumeID, "/")
+	volumeID = split[len(split)-1]
+
+	// ebsnvme-id is present on AWS-provided Windows Server AMIs
+	// https://docs.aws.amazon.com/AWSEC2/latest/WindowsGuide/nvme-ebs-volumes.html#identify-nvme-ebs-device
+	ebsnvmeID := `C:\ProgramData\Amazon\Tools\ebsnvme-id.exe`
+	exec := attacher.host.GetExec(awsElasticBlockStorePluginName)
+	output, err := exec.Run("powershell", "/c", ebsnvmeID)
```

Comment:
why does this need PowerShell? It seems like you could call it directly and save the few seconds needed for PowerShell to start up.

Reply:
I'm not sure, I'm cargo-culting other code: https://github.com/kubernetes/kubernetes/blob/ed239cefa6b3add3d53d55e02fa9c28a0bbfc25a/pkg/volume/azure_dd/azure_common_windows.go#L34

Reply:
@andyzhangx any ideas why we should invoke commands using `powershell /c` vs directly? Thanks

Reply:
Ah, well in the above case a powershell cmdlet is used. In my case I'm just running an .exe so there's no need to go through powershell.

Reply:
Good catch on this.